mvcc/backend/backend.go

```diff
@@ -65,6 +65,8 @@ type Backend interface {
 	// Since the backend can manage free space in a non-byte unit such as
 	// number of pages, the returned value can be not exactly accurate in bytes.
 	SizeInUse() int64
+	// OpenTxN returns the number of currently open read transactions in backend.
+	OpenTxN() int64
```

Comment:
OpenReadTxN

Reply:
I got the name from bolt.Stats.OpenTxN, but really wanted to change the name to what you are suggesting. So I will rename:)